testdata/kubeconfig-proxy-url.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
```

Comment:
Just out of curiosity, why is this file so different from the same file that was added in #2091? Is it because this version seems to have a bit more testing?

Reply:
Yes, I added more tests to address all options in for proxy-url. This is mainly to check if agent types are correct.